caas/kubernetes/provider/klog.go

```diff
@@ -30,9 +36,14 @@ func (k *klogAdapter) Enabled() bool {
 func (k *klogAdapter) Error(err error, msg string, keysAndValues ...interface{}) {
 	if err != nil {
 		k.Logger.Errorf(msg+": "+err.Error(), keysAndValues...)
-	} else {
-		k.Logger.Errorf(msg, keysAndValues...)
+		return
+	}
+
+	if strings.HasPrefix(msg, klogPrefixIgnoreProxy) {
+		k.Logger.Debugf(msg, keysAndValues)
```

Comment:
Should this be `keysAndValues...` like the existing call?

Reply:
I don't think so